README.md

```diff
@@ -112,6 +112,26 @@ This means an adaptor is not available to download from Prebid.org as soon as th
 
 * [jekyll - check for non empty](https://michaelcurrin.github.io/dev-cheatsheets/cheatsheets/jekyll/liquid/conditionals/non-empty.html)
 
+## Codex
```

Comment:
This great, thanks 🙏 

Hopefully those agents can soon spin-up the devcontainer, which should make this unnecessary in the future